modules/tinymce/src/core/main/ts/newline/InsertNewLine.ts

```diff
@@ -15,35 +15,39 @@ import * as InsertBr from './InsertBr';
 import * as NewLineAction from './NewLineAction';
 
 const insert = (editor: Editor, evt?: EditorEvent<KeyboardEvent>) => {
-  NewLineAction.getAction(editor, evt).fold(
+  const insertBlock = (stepUntilFinding?: string) =>
     () => {
       if (Type.isNonNullable(evt)) {
-        const event = fireFakeBeforeInputEvent(editor, 'insertLineBreak');
+        const event = fireFakeBeforeInputEvent(editor, 'insertParagraph');
         if (event.isDefaultPrevented()) {
           return;
         }
       }
 
-      InsertBr.insert(editor, evt);
+      InsertBlock.insert(editor, evt, stepUntilFinding);
```

Comment:
Weird why are these flipped from InsertBr to InsertBlock on the adt the new case `newBlock` is added after the block so that shouldn't affect the ordering. This fold should still be (br, block, newBlock, none) maybe I'm not reading the code right here?

Reply:
This is just git being confused when showing the differences. Since the two versions of InsertBlock were almost identical I moved the function out to a variable, but git then confuses it with the InsertBr-function that was first. The InsertBr-function is unchanged.

Reply:
Ahh right yes when I view the whole file it looks correct. Thanks make sense now.